### DISCUSSION TITLE
Welcome to nscp Discussions!

### DISCUSSION THREAD

**Discussion by mickem** [mickem/nscp#764] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Ask for your favorite feature
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by willemdh** [comment]
Thanks @mickem for all the effort you put in NSClient++!  Good to see some movement in the nscp repo again... 🥇

**Comment by sysshob** [comment]
Michael, thank you so much for such a great product.
My next challenge - running NSClient++ on RHEL 7. I hope it can be done.

**Comment by rsudre** [comment]
@mickem  nsclient++ web interface accepts the password, but does not leave the login screen. Could anyone help me how to solve it?

nsclient version nscp-00.5.41-x64 installed on windows 2008server.

Thanks!
![image](https://github.com/mickem/nscp/assets/32674090/79bb916b-26cd-4bf4-a878-3a53db0edc37)

**Reply by RSI-OPENC** [reply]
Same issue for me here. Password is a good one otherwise an error appears. It look like loop!

**Reply by mickem** [reply]
Web interface is a "Known issue" in the latest build. It was built with an old version of vue and updating that is harder than making a new one, so the plan is to replace it with a newer on during summer...

**Reply by RSI-OPENC** [reply]
Hi @mickem ,
Great, thanx for the answer. Is there a workaround in the meantime (roolback to previous version)?
To be clear, I d'ont know what the HTML UI offer, is that necessary to manage the monitor connector ?
For test only, I try to connect my centreon to my local Win11 with Centreon NSClient++, is that possible and compatible or only SNMP is available out of Windows server versions. You point of view would be welcome.

**Reply by mickem** [reply]
The WEB UI allows you to "configure things" so if you manage configuration via the config file you should not notice anything.
All the APIs are still accessible and should work so it is only the web front end...

So my thinking (why I released without it) is that it is not really an essential part but a convienience...

**Reply by RSI-OPENC** [reply]
Great thanx a lot for your time. That's really clear. .ini is ok and well describe.
Is there a doc to show API interface?
I'd like to make a call with postman, just to see if API give answer
I found that one but am quite confuse with : https://docs.centreon.com/fr/pp/integrations/plugin-packs/getting-started/how-to-guides/troubleshooting-plugins/

**Reply by mickem** [reply]
Sure docs for the REST API can be found here: https://nsclient.org/docs/api/rest/ (see side menu for various modules)

**Reply by mickem** [reply]
Example for running a query for instance: https://nsclient.org/docs/api/rest/queries/#example_1

**Reply by RSI-OPENC** [reply]
I can see answer. Great. I've now to look centreon supervisor.
Thanx a lot
